packages/ckeditor5-list/docs/features/installation.md

```diff
@@ -0,0 +1,104 @@
+---
+menu-title: Installation
+meta-title: Lists installation | CKEditor 5 Documentation
+category: features-lists
+order: 40
+modified_at: 2024-01-02
+---
+
+# Installation
+
+<info-box info>
+	The lists feature is enabled by default in all {@link installation/getting-started/predefined-builds predefined builds}. The installation instructions are for developers interested in building their own, custom rich text editor.
+</info-box>
```

Comment:
Don't we want to remove the info box with a link to predefined builds?

Reply:
Only after Kowalski lands I believe.